baremetal/metal3machine_manager.go

```diff
@@ -208,13 +210,13 @@ func (m *MachineManager) SetPauseAnnotation(ctx context.Context) error {
 		host.Annotations = make(map[string]string)
 		host.Annotations[bmh.PausedAnnotation] = pausedAnnotationKey
 	}
-	return updateObject(m.client, ctx, host)
+	return helper.Patch(ctx, host)
 }
 
 // GetBaremetalHostID return the provider identifier for this machine
 func (m *MachineManager) GetBaremetalHostID(ctx context.Context) (*string, error) {
 	// look for associated BMH
-	host, err := m.getHost(ctx)
+	host, _, err := m.getHost(ctx)
```

Comment:
is blank identifier replacing `helper` ?

Reply:
yes, because helper is unused here